internal/sys/fs.go

```diff
@@ -65,15 +65,26 @@ var emptyFSContext = &FSContext{
 	lastFD:      2,
 }
 
-// NewFSContext returns a mutable context if the fs is not EmptyFS.
+// NewFSContext returns a mutable context if the fs is not EmptyFS. The file
+// descriptor used in the case of non-empty is 3.
```

Comment:
```suggestion
// descriptor for the always-open root file, `/`, is 3.
```

The file descriptor of what? is what I wondered when reading the comment so tried to clarify.

Though I didn't use the word preopened here and went with always-open - I had never known the concept of preopened files until diving deep into these functions, but basically it seems file system roots are all preopened, and any other path isn't. "always-open root file" seems to convey both the semantics and the concept, maybe

Reply:
the only parameter is `fs`, so I think I'll tweak the doc a bit further.

It should be clear and unambiguous that the only parameter here is the file system and so the only file descriptor we are talking about is what that file system becomes. Good point it is root and maybe the param should be renamed as such.

meanwhile this is internal docs. The actual parameter is described in ModuleConfig